frontend/dogplus-frontend/src/types/user.ts

```diff
@@ -7,6 +7,13 @@ export interface UserData {
   isApproved?: boolean;
 }
 
+export interface User {
+  id: string;
+  username: string;
+  email: string;
```

Comment:
Is there a need to add a new interface for user instead of just updating the previous interface? And also we should remove password from the UserData interface